README.md

```diff
@@ -32,3 +32,116 @@ zopen build
 Note that running `zopen build` against your development repo will NOT install the code into your
 environment.
 
+
+# Developer Notes
+
+## workflow
+
+Say you want to work on meta
+
+### update existing meta
+
+This will update the ~/zopen/usr/local/bin versions of the system binaries as part
+of the meta package.
+
+```
+$ zopen upgrade meta -y
+```
+
+### Create the work dev repos for meta and metaport
+
+This sets up the repos for work.  Note this workflow
+requires meta and metaport dirs to be named meta
+and metaport respectively.
+
+```
+$ cd ~/zopen/dev
+$ git clone git@github.com:ZOSOpenTools/meta.git
+$ git clone git@github.com:ZOSOpenTools/metaport.git
+```
+
+### Create a branch in the dev version of meta for work
+
+
+```
+$ cd ~/zopen/dev/meta
+$ git branch johndev
+$ git checkout johndev
+```
+
+### Create an environment script to setup for dev work
+
+
+
+```
+$ cd ~/zopen/dev
+$ cat << EOF > setenv.sh
+> export ZOPEN_META_DEV_ROOT=$HOME/zopen/dev/meta
+> export ZOPEN_META_BRANCH=johndev
+> EOF
+
+
+$ cat setenv.sh
+export ZOPEN_META_DEV_ROOT=/z/jd895801/zopen/dev/meta.official
```

Comment:
change to $HOME/zopen/dev/meta

Reply:
You are welcome.   Yes, indeed, I'll try to merge the changes into the original so it flow better.